admin-console/src/main/webapp/src/pages/BundlePage.tsx

```diff
@@ -119,17 +129,18 @@ export const BundlePage: React.FunctionComponent = () => {
                             <ToolbarContent>
                                 <ToolbarItem>
                                     <Button variant='primary' type='button' onClick={ createApplication }> Create Application </Button>
-                                    <CreateEditApplicationModal
+                                    {showModal && <CreateEditApplicationModal
```

Comment:
This removes the component when is hidden - This is made so that the state is cleared each time we show/hide.